app/controllers/redirections_controller.rb

```diff
@@ -18,8 +18,7 @@ def previous
 
   def find_or_create_redirection
     Redirection.find_by(slug: params[:slug]) ||
-      # Creation paused for now as we sort out weirdness
-      # RedirectionCreation.perform(referrer, params[:slug]) ||
+      RedirectionCreation.perform(referrer, params[:slug]) ||
```

Comment:
Should we put this behind any kind of flag/env var in case we want to re-enable it quickly?

Reply:
Excellent point. Fixed in 6198d33a93fc17853d43898d572da397bf8ce7a9.